<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>431</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="info">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
       <widget class="QPushButton" name="change_space">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Change space</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="Test1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Test1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pTest2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Test2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="spaces">
      <property name="styleSheet">
       <string notr="true">background-color:silver;</string>
      </property>
      <widget class="QWidget" name="RPG_space">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>201</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QListWidget" name="QuestList">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="AddingQuest">
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
             <property name="text">
              <string>Add Quest</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="character_icon">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>199</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Це твій персонаж</string>
          </property>
          <property name="whatsThis">
           <string>Це твій персонаж</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QFrame" name="active_item">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>398</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="Open_inventory">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>380</y>
             <width>131</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Open inventory</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>131</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Active Item</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QFrame" name="infoCharacter">
          <property name="minimumSize">
           <size>
            <width>136</width>
            <height>201</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLabel" name="ManaLabel">
             <property name="styleSheet">
              <string notr="true">color:black;
</string>
             </property>
             <property name="text">
              <string>Mana</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="LevelLabel">
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="HealthLabel">
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
             <property name="text">
              <string>Health</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="ExperienceLabel">
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
             <property name="text">
              <string>Exp</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="more_characteristics">
             <property name="styleSheet">
              <string notr="true">color:black;</string>
             </property>
             <property name="text">
              <string>More</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Note_space">
       <widget class="QTextEdit" name="TextNote">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>451</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;
color:black;
</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="TitleNote">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>441</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white;
color:black;</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listTag">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="NoteTime">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>25</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tags_option">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>31</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:black;
background-color:rgb(129, 255, 134);</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>change_space</tabstop>
  <tabstop>Test1</tabstop>
  <tabstop>pTest2</tabstop>
  <tabstop>QuestList</tabstop>
  <tabstop>AddingQuest</tabstop>
  <tabstop>Open_inventory</tabstop>
  <tabstop>more_characteristics</tabstop>
  <tabstop>TextNote</tabstop>
  <tabstop>TitleNote</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
